packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -1661,6 +1726,14 @@ export default function<T, P, I, TI, HI, PI, C, CC, CX, PL>(
             // There's no time left. Mark this root as complete. We'll come
             // back and commit it later.
             root.finishedWork = finishedWork;
+
+            if (enableProfileModeMetrics) {
+              // If we didn't finish, pause the "actual" render timer.
+              // We'll restart it when we resume work.
+              if (nextUnitOfWork !== null) {
```

Comment:
Same question as the other spot. Is this check needed or can we just always call pauseActualRenderTimerIfRunning?